src/commands/service/create.ts

```diff
@@ -173,9 +171,13 @@ export const createService = async (
   shelljs.mkdir("-p", newServiceDir);
 
   // Create azure pipelines yaml in directory
-  await generateStarterAzurePipelinesYaml(rootProjectPath, newServiceDir, {
-    variableGroups: values.variableGroups
-  });
+  generateServiceBuildAndUpdatePipelineYaml(
+    rootProjectPath,
+    values.ringNames,
+    values.displayName ? values.displayName : serviceName, // displayName takes priority over serviceName (which could be '.')
```

Comment:
i know in reconcile we try to use displayName and ignore services that are named '.' - can we catch and do something about a service name being '.' ?

Reply:
That entry check will be done with this ticket: https://github.com/microsoft/bedrock/issues/904

But I'll also add a boolean check and throw an error.